cryptic-game/frontend
# Fixed history command display order

**Description**
The history command now no longer uses `.reverse()` on the history.
It now goes through the list in reverse via a for loop.

**Testing Instructions**
1. Open a terminal
2. Enter the command `help 1`
3. Enter the command `help 2`
4. Enter the command `history`
5. Enter the command `history` again
6. After step 5 you should see the history of commands in the right order (first help 1, then help 2).

**Additional Notes**
This was the easiest implementation that doesn't reverse the list again or copies it.
Changing the order in which the commands are stored would require changing the code for going through the history with the arrow keys in a non-trivial way that would make it easier to introduce bugs there.
